src/app/comments/newCommentForm/NewCommentForm.jsx

```diff
@@ -48,7 +50,11 @@ class NewCommentForm extends Component {
   }
 
   render() {
-    const { locationData, isPostingComment } = this.props;
+    const { locationData, isPostingComment, locationError } = this.props;
+
+    if(locationError){
```

Comment:
I think we have an open issue for this https://github.com/streetlives/streetlives-web/issues/113

Reply:
Just checked: master has *42* errors, and basically of only 2 types: either jsx-a11y or no-extraneous-dependencies (those 2 types account for 40 of the 42). They affect *16* files in total.

In this branch, there are currently *243* errors, affecting *49* files.

Reply:
So, while I tend to agree that 42 is also a lot, the fact there's currently practically only 2 types means they could be solved more or less at once, I hope.
Introducing 200 new errors of various kinds, however, I think is a bigger problem.

Therefore, I stand by my opinion that it would be best to first fix the linter (whether the existing 42 errors are also fixed or not), and only then merge a PR that touches so many files.